packages/utils/src/bootstrap.ts

```diff
@@ -1,9 +1,11 @@
 import Multiaddr from 'multiaddr'
 import dns from 'dns'
+import semver from 'semver'
 
 const FULL_VERSION = require('../package.json').version
+const cleanVersion = (version) => `${semver.major(version)}.${semver.minor(version)}.${semver.patch(version)}`
```

Comment:
I don't think we need a full dependency for this - it's literally a slice on an array.

Reply:
See MAJ_MIN in core.

Reply:
Ah, let me see

Reply:
Updating, resolving conversation.